.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Automatic Publish
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  PAT: ${{ secrets.PAT }}
```

Comment:
What is `PAT`?

Reply:
https://code.visualstudio.com/api/working-with-extensions/publishing-extension#get-a-personal-access-token

I have added this link to `.envrc_sample`

Reply:
+1, I'd probably rename this to `VSCODE_ACCESS_TOKEN`. Would be more clear in like a year from now, if someone just looks at the code.

Reply:
`AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN`?

Reply:
Going with `AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN` then! Updating shortly!

Reply:
Done